extensions/wikia/Forum/ForumSpecialController.class.php

```diff
@@ -113,11 +120,13 @@ public function createNewBoardModal() {
 		if ( !$this->wg->User->isAllowed( 'forumadmin' ) ) {
 			$this->displayRestrictionError();
 			wfProfileOut( __METHOD__ );
+
 			return false;
 			// skip rendering
 		}
 
-		$this->setVal( 'title', wfMessage( 'forum-admin-create-new-board-modal-heading' )->plain() );
+		$this->setVal( 'title',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)